Engine/ILockable.cs

```diff
@@ -82,11 +82,14 @@ public interface ILockManager : ITapPlugin
 
     internal class LockManager
     {
-        private List<ILockManager> managers;
+        private ILockManager[] managers;
 
         public LockManager()
         {
-            managers = PluginManager.GetPlugins<ILockManager>().OrderByDescending(t => t.GetDisplayAttribute().Order).Select(t => (ILockManager)t.CreateInstance()).ToList();
+            managers = PluginManager.GetPlugins<ILockManager>()
+                .OrderByDescending(t => t.GetDisplayAttribute().Order)
+                .Select(t => (ILockManager)t.CreateInstance())
```

Comment:
Can we use `TrySelect` here? CreateInstance can throw when plugins cannot be loaded for some reason, such as when a plugin depends on an incompatible runtime.